gridcapa-task-manager-app/src/main/resources/application.yml

```diff
@@ -71,3 +71,7 @@ stomp:
 logback:
   amqp:
     exchange: task-manager-logs-exchange
+
+purge-task-events:
+  cron: 0 0 0 * * *
```

Comment:
i think you have an extra 0 here if your intention is to run the job every day at midnight. 
Plus i think midnight is not the best time do give task manager extra work. I think we must choose a time where the server is less busy.

Reply:
Cron for linux (yaml files for example) have 5 arguments, but for scheduling in java spring there are seconds first, so 6 arguments : https://stackoverflow.com/questions/30887822/spring-cron-vs-normal-cron

I can change the hour, 5AM for example ?